independent-publisher-connectors/IBM Watson Text to Speech/apiProperties.json

```diff
@@ -0,0 +1,66 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "url": {
+        "type": "string",
+        "uiDefinition": {
+          "displayName": "Service URL",
+          "description": "The service URL",
+          "tooltip": "Provide your service URL",
+          "constraints": {
+            "tabIndex": 1,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      },
+      "username": {
+        "type": "string",
+        "uiDefinition": {
+          "displayName": "API Key Name",
+          "description": "The API key name",
+          "tooltip": "Provide the username",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true",
+            "allowedValues": [
+              {
+                "text": "Default",
+                "value": "apikey"
+              }
+            ]
+          }
+        }
+      },
+      "password": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The service API key",
+          "tooltip": "Provide the API Key",
+          "constraints": {
+            "tabIndex": 3,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#0F62FE",
+    "scriptOperations": [
+      "Synthesize"
+    ],
+    "capabilities": [],
+    "policyTemplateInstances": [
+      {
+        "templateId": "dynamichosturl",
+        "title": "SetHost",
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "@connectionParameters('url')"
+        }
+      }
+    ],
+    "publisher": "Lucas Titus"
```

Comment:
@lucastitus please add the stackOwner property. 
"stackOwner": "Name of company or organization that connector is connecting to."